tests/unit/test_jlink.py

```diff
@@ -6260,6 +6260,36 @@ def test_set_log_file_raises_exception_if_SetLogFile_fails(self):
         with self.assertRaises(JLinkException):
             self.jlink.set_log_file('my/file/path')
 
+    def test_set_script_file_success(self):
+        """Tests that set_script_file uses provided parameters.
+
+        Args:
+          self (TestJLink): the ``TestJLink`` instance
+
+        Returns:
+          ``None``
+        """
+        args = ['my/file/path']
+        expected = ['scriptfile = my/file/path']
+        self.jlink.exec_command = mock.Mock()
+        self.jlink.exec_command.return_value = 0
+
+        self.jlink.set_script_file(*args)
+        self.jlink.exec_command.assert_called_once_with(*expected)
+
+    def test_set_script_file_raises_exception_if_ExecCommand_fails(self):
```

Comment:
```suggestion
    def test_set_script_file_raises_exception_if_exec_command_fails(self):
```
Python style is `snake_case` for function names, so we should rename `ExecCommand -> exec_command`.
- [ ] Fix function name.

Reply:
I went with the camel case because it felt more consistent with the names of the other tests in that file, particularly those towards the end such as: 
- `test_set_log_file_raises_exception_if_SetLogFile_fails()`
- `test_cp15_register_write_raises_exception_if_CP15_WriteEx_fails()`

But snake case is PEP8 standard, so I'm happy to accept the change.  Your call.